cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiffTest.java

```diff
@@ -37,6 +39,17 @@
     KafkaVersion kafkaVersion = VERSIONS.version(KAFKA_VERSION);
     private int brokerId = 0;
 
+    // use reflection to instantiate ConfigEntry
+    private Constructor constructor;
+    {
+        try {
+            constructor = ConfigEntry.class.getDeclaredConstructor(String.class, String.class, ConfigEntry.ConfigSource.class, boolean.class, boolean.class, List.class, ConfigEntry.ConfigType.class, String.class);
+            constructor.setAccessible(true);
+        } catch (NoSuchMethodException e) {
+            fail();
+        }
+    }
```

Comment:
There doesn't seem to be any benefit from having this as a field. I would just factor out a `createConfigEntry` method which did the `getDeclaredConstructor` and `newInstance` in one place.